experimenter/experimenter/nimbus-ui/src/components/PageHome/index.test.tsx

```diff
@@ -49,6 +49,7 @@ describe("PageHome", () => {
       ["drafts", screen.getByText("Draft (3)")],
       ["live", screen.getByText("Live (3)")],
       ["archived", screen.getByText("Archived (1)")],
+      ["owned", screen.getByText("My Experiments (1)")],
```

Comment:
Is it potentially confusing to call this "owned" when it's owned and subscribed?

Reply:
It isssssss but trying to use more words there or different words there didn't feel any better, but I'm open to suggestions.

Reply:
Yea fair enough, and obviously not a big deal. I was trying to think of a suggestion when I asked but didn't come up with anything obvious. Would just "subscribed" be better if we want one word? I think it's more implicit that "subscribed" includes "owned" but not necessarily the other way around.